update virtual study model

kalletlak opened the pull request.

kalletlak committed:
update virtual study model

zhx828 commented on app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js:
```diff
@@ -154,18 +154,18 @@
                       });
                       self_.selectedSamplesNum = _selectedSamplesNum;
-                      self_.selectedPatientsNum = _selectedPatientsNum;
+                      self_.selectedPatientsNum = _selectedSamplesNum;
```
# of patients always matches with # of samples?

kalletlak replied:
Sorry! by mistake changed it.
 btw where are `selectedSamplesNum` and `selectedPatientsNum` used here?

zhx828 commented on app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js:
```diff
@@ -154,18 +154,18 @@
-                      self_.selectedPatientsNum = _selectedPatientsNum;
+                      self_.selectedPatientsNum = _selectedSamplesNum;
 
+                      console.log(self_.stats)
```
No longer needed.

kalletlak committed:
remove patient count from description

cleanup patient count as this information is never used and samples
would be sufficient to retrieve this information

kalletlak committed:
cleanup

and port changes from https://github.com/cBioPortal/cbioportal/pull/3344
